modules/mono/glue/GodotSharp/GodotSharp/Core/Vector2I.cs

```diff
@@ -504,15 +504,15 @@ public Vector2I(int x, int y)
         }
 
         /// <summary>
-        /// Converts a <see cref="Vector2"/> to a <see cref="Vector2I"/>.
+        /// Converts a <see cref="Vector2"/> to a <see cref="Vector2I"/> by truncating
+        /// components' fractional parts (rounding towards zero). For a different
+        /// behavior consider passing the result of <see cref="Vector2.Ceil"/>,
+        /// <see cref="Vector2.Floor"/> or <see cref="Vector2.Round"/> to this constructor instead.
```

Comment:
This isn't a constructor, it's a conversion operator. I guess this is just a copy-paste mistake, the other types have the same issue.

Reply:
> I guess this is just a copy-paste mistake

Yeah, I've missed that part. Corrected!